lamindb/examples/__init__.py

```diff
@@ -7,8 +7,9 @@
    datasets
    cellxgene
    croissant
+   mlflow
 
 """
 
-from . import croissant, datasets, schemas
+from . import croissant, datasets, ml_tracking, schemas
```

Comment:
For consistency with the `integrations` module and `croissant` and `cellxgene`, can we call this `mlflow` and `wandb` rather than introducing an ad-hoc grouping?

If you want to create a new namespace, then I'd say `mlops` for consistency with `lamin-mlops`. Danger is that this might change whereas `mlflow` and `wandb` won't change. ☺️